docs/docs/reference/Stemming.md

```diff
@@ -6,14 +6,65 @@ description: >
     Stemming support
 ---
 
-# Stemming Support
+# Stemming and Multi-language Support
 
-RediSearch supports stemming - that is adding the base form of a word to the index. This allows the query for "going" to also return results for "go" and "gone", for example.
+RediSearch supports stemming - that is adding the base form of a word to the index. This allows the query for "`going`" to also return results for "`go`" and "`gone`", for example.
```

Comment:
I suggest to change the example, because the [snowball demo](https://snowballstem.org/demo.html), does not detect that ``gone`` steams from ``go``. Maybe because it's an irregular verb. With a regular verb (``hire, hiring, hired``), or common endings (``relate, relational, related``) it works well

Reply:
```suggestion
RediSearch supports stemming - that is adding the base form of a word to the index. This allows the query for "`hiring`" to also return results for "`hire`" and "`hired`", for example.
```